File: LegacyCodeWithoutTest/SchedulerDotNet/TimeServices.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SchedulerDotNet
{
    public class TimeServices
    {

        public static bool IsPastDue(DateTime date)
        {

            return date.CompareTo(DateTime.Now) < 0;
        }

        public static bool IsWorkDay(DateTime date)
        {
            return false;
        }

        public static bool IsHoliday(DateTime date)
        {
            return false;
        }
    }
}

File: schedulerJava/src/main/java/TimeServices.java
import java.util.Date;

public class TimeServices {

	public static boolean isPastDue(Date date) {
		
		return date.before(date);
	}
	
	public static boolean isWorkDay(Date date) {
		return false;
	}
	
	public static boolean isHoliday(Date date) {
		return false;
	}
}
